systemd/system/cri-resource-manager-agent.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=CRI Resource Manager Node Agent
+Documentation=https://github.com/intel/cri-resource-manager
+Before=cri-resource-manager.service
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/sysconfig/cri-resource-manager
+ExecStart=/usr/bin/cri-resmgr-agent --kubeconfig $KUBE_CONFIG
```

Comment:
better not to do that. agent should be run as daemonset, not directly on the host. it is ok for debug/development purposes, but in production environment it should be daemonset with incluster config, instead of directly passed config.

Reply:
Ok, fair enough. How about if I separate it out to sub-package of its own ? Then it's not installed by default but is available and built from the same sources if someone wants/needs it on the host side ?